test/Dapper.FluentMap.Dommel.Tests/Dapper.FluentMap.Dommel.Tests.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net461;netcoreapp2.0;netcoreapp2.1;netcoreapp3.0</TargetFrameworks>
+    <TargetFrameworks Condition=" '$(OS)' != 'Windows_NT' ">netcoreapp2.0;netcoreapp2.1;netcoreapp3.0</TargetFrameworks>
```

Comment:
Only target .NET Core 3.0 (or preferably 3.1).

Reply:
Would prefer to leave tests in there since all those versions should be supported and the release is only for .net standard 2.0

Reply:
There isn't really much value in multiple target frameworks, as they all use the same netstandard binary. Let's only use netcoreapp3.1 for simplicity.